.github/workflows/ci.yml

```diff
@@ -1,41 +1,55 @@
+---
 name: Erlang CI
 
-on: [push, pull_request]
+"on": [push, pull_request]
 
 jobs:
 
   build:
 
-    runs-on: ubuntu-20.04
+    runs-on: ${{matrix.os}}
 
     strategy:
       matrix:
-        otp: ['23.3', '24.3', '25.0']
-        rebar: ['3.18.0']
+        include:
+          - otp: 23
+            rebar: 3.18
+            os: ubuntu-20.04
+          - otp: 24
+            rebar: 3.22
+            os: ubuntu-22.04
+          - otp: 25
+            rebar: 3.22
+            os: ubuntu-22.04
+          - otp: 26
+            rebar: 3.22
+            os: ubuntu-22.04
```

Comment:
Adds 26, without moving minimum of 23, which forces for a specific `rebar3` and Ubuntu...

Reply:
We can totally drop support for OTP23

Reply:
60743a3